tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -943,12 +946,12 @@ TEST_F(ValidationTest, QuadOpInVS) {
       "struct PerThreadData { int "
       "input; int output; }; RWStructuredBuffer<PerThreadData> g_sb; "
       "void main(uint vid : SV_VertexID)"
-      "{ g_sb[vid].output = WaveActiveSum(g_sb[vid].input); }",
+      "{ g_sb[vid].output = WaveActiveBitAnd((uint)g_sb[vid].input); }",
```

Comment:
Why was this change needed?

Reply:
The IR modification resulted in an invalid signature for the target operation, which would now be caught, causing assert and preventing this test from working.  The operation was invalid because it had the wrong number of operands.  I chose a different intrinsic so the number of operands would be the same, and IR modification would result in valid IR.